scripts/build.py

```diff
@@ -1109,6 +1118,42 @@ def _verify_build(
             THIRD_PARTY_GENERATED_OUT_DIR, third_party_css_final_filename)])
 
 
+def _verify_file_count(
+        input_dirnames, output_dirnames):
+    """Verify number of files between source directory and final directory
+        matches after build process finishes.
+
+        Args:
+            input_dirnames: list(str). List of directory paths that contain
+                source files.
+            output_dirnames: list(str). List of directory paths that contain
+                built files.
+    """
```

Comment:
Missing Raises: docstring

Reply:
No exception is raised by this function, it is raised by compare file count function which has the docstring.

Reply:
True, but the whole point of this function (semantically) is to raise an exception if there's a problem, so you might want to document it in this case.

Reply:
This function is now removed.